docs/add-a-browser.md

```diff
@@ -0,0 +1,49 @@
+# Add a browser setting to Firefox Lockbox
+
+_These instructions are for developers wanting to add another browser setting so when users tap to open an entry's URL it will be sent to the web browser of their choice._
+
+
+1. Find and add the query scheme the new browser has registered to `LSApplicationQueriesSchemes` in `Common/Resources/Info.plist`. For example we'll use [DuckDuckGo](https://github.com/mozilla-lockbox/lockbox-ios/compare/master...joeyg:duckduckgo?expand=1):
+
+  ```
+  <string>ddgQuickLink</string>
+  ```
+
+
+2. Then add a new case to `PreferredBrowserSetting` in `ExternalLinkAction.swift`:
+
+  ```swift
+  case DuckDuckGo
+  ```
+
+3. Define that new case in the `getPreferredBrowserDeeplink` switch statement and return the string (with that new query scheme as the expected URL syntax):
```

Comment:
developers might need to URL-encode the URL that they're sending here. I know there's some trickiness here with how Firefox for iOS, at least, expects to receive URLs, so maybe include a note about "test deeplinks to make sure that they open as expected" and URL-encoding...